modules/candidate_parameters/test/TestPlan.md

```diff
@@ -67,4 +67,18 @@
         * Error: date cannot be later than today
     * Consent to Study = No (changing/updating an already existing 'Yes' consent); Date (Withdrawal) of Consent = valid random date; Confirmation (Withdrawal) Date of Consent = same random date
         * No error
-        * Make sure they update properly in the front-end and backend
\ No newline at end of file
+        * Make sure they update properly in the front-end and backend
+
+### Date of Birth Tab
+32. Check that date of birth can only be amended if user has candidate_dob_edit permission.
```

Comment:
does the tab show to the user if they dont have the permission ?

Reply:
yes, however the field is disabled.